src/app/components/import/AttachedFile.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Icon, Button } from 'react-components';
+
+import humanSize from 'proton-shared/lib/helpers/humanSize';
+
+const splitExtension = (filename) => {
+    if (!/\./.test(filename)) {
+        return [filename, ''];
+    }
+    const ext = filename.split('.').pop();
+    const name = filename.substring(0, filename.length - ext.length - 1);
+
+    return [name, ext];
+};
+
+const AttachedFile = ({ file, iconName, className, clear, onClear, ...rest }) => {
+    return (
+        <div className={`flex w100 ${className}`} {...rest}>
```

Comment:
Some utility function to handle classNames could be better (most popular is `classnames`). Also className is not mentioned on propTypes

Reply:
Again sticking to the way we're doing things in the other React projects. But yeah, one could do it differently.